src/univariate/continuous/normal.jl

```diff
@@ -107,6 +107,9 @@ Base.:*(c::Real, d::Normal) = Normal(c * d.μ, abs(c) * d.σ)
 
 rand(rng::AbstractRNG, d::Normal{T}) where {T} = d.μ + d.σ * randn(rng, float(T))
 
+rand!(rng::AbstractRNG, d::Normal, A::AbstractArray{<:Real}) = 
+    A .= randn!(rng, A) .* d.σ .+ d.μ
```

Comment:
I wonder if it would be even better to use `muladd` here:
```suggestion
rand!(rng::AbstractRNG, d::Normal, A::AbstractArray{<:Real}) = A .= muladd.(d.σ, randn!(rng, A), d.μ)
```

Reply:
Seems to be a tiny bit faster in my benchmarks:
```julia
julia> using BenchmarkTools, Distributions

julia> n = 1_000;

julia> @btime randn($n);
  1.632 μs (1 allocation: 7.94 KiB)

julia> @btime rand($(Normal()), $n);
  3.692 μs (1 allocation: 7.94 KiB)

julia> f(d::Normal, n::Int) = d.μ .+ d.σ .* randn(n)
f (generic function with 1 method)

julia> @btime f($(Normal()), $n);
  2.296 μs (2 allocations: 15.88 KiB)

julia> @btime f($(Normal()), $n);
  2.282 μs (2 allocations: 15.88 KiB)

julia> g(d::Normal, n::Int) = muladd.(d.σ, randn(n), d.μ)
g (generic function with 1 method)

julia> @btime g($(Normal()), $n);
  2.248 μs (2 allocations: 15.88 KiB)

julia> @btime g($(Normal()), $n);
  2.244 μs (2 allocations: 15.88 KiB)
```